api/metrics-api-jaxrs-1.1/pom.xml

```diff
@@ -124,6 +124,16 @@
   <build>
     <finalName>hawkular-metrics-api-rest-1.1</finalName>
     <plugins>
+      <plugin>
```

Comment:
I will remove the skip property from the config, but leave the management port in case, users want to override that from the maven command line.